test/test_testvectors.py

```diff
@@ -10,28 +10,27 @@
 
 
 def test_testvectors():
+    @helpers.filtered_test
+    def check_testvectors(implementation):
```

Comment:
Why did you move this inside?

Reply:
To have it not be detected by nosetests as a test, because test is part of the name of this function. It used to be called `check_vectors`, but as the decorator filter uses the name of the function to determine if it should be run.